docs/samples/modelmesh_sdk_sample.ipynb

```diff
@@ -0,0 +1,562 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Sample for ModelMesh Serving using the KServe Python SDK\n",
+    "\n",
+    "## This notebook shows how to use the KServe SDK to create, update, and delete an InferenceService\n",
+    "\n",
+    "- This sample assumes ModelMesh Serving was deployed using the [quickstart guide](https://github.com/kserve/modelmesh-serving/blob/main/docs/quickstart.md)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from kubernetes import client\n",
+    "from kserve import constants\n",
+    "from kserve import V1beta1InferenceService\n",
+    "from kserve import V1beta1InferenceServiceSpec\n",
+    "from kserve import V1beta1PredictorSpec\n",
+    "from kserve import V1beta1SKLearnSpec\n",
+    "from kserve import V1beta1TFServingSpec\n",
+    "from kserve import V1beta1StorageSpec\n",
+    "from kserve import KServeClient"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### The variables being set are `namespace`, `name`, `protocol_version`\n",
+    "- The `namespace` definition is where the InferenceService will be deployed to\n",
+    "- `name` will be the name of the InferenceService\n",
+    "- For ModelMesh, the `v2` protocol must be used since it doesn't support the default `v1` protocol"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "namespace = 'modelmesh-serving'\n",
+    "name='mnist-sample'\n",
+    "protocol_version='v2'"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Define InferenceService specifying ModelMesh as the deploymentMode"
```

Comment:
maybe we can add a line here to describe why that is important?

> While both the KServe controller and ModelMesh controller will reconcile InferenceService resources, …
the ModelMesh controller will only handle those InferenceServices with the [serving.kserve.io/deploymentMode](http://serving.kserve.io/deploymentMode): ModelMesh annotation. Otherwise, the KServe controller will handle reconciliation.
the KServe controller will not reconcile an InferenceService with the [serving.kserve.io/deploymentMode](http://serving.kserve.io/deploymentMode): ModelMesh annotation, and will defer under the assumption that the ModelMesh controller will handle it.